MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -19,16 +22,21 @@ def store_on_fs(data, file_name):
 	"""
 	Store data in file named `file_name`
 	"""
-	with open(file_name, "w") as f:
-		f.write(data)
+	if data:
+		try:
+			with open(file_name, "w") as f:
+				f.write(str(data))
+		except:
+			return
```

Comment:
Can we print the exception?

Reply:
@llxia it would be great if you could explain me this a bit more.
Like if there aren't any quotations in a particular file, it stores a `[]` in the new file that would be created in the IssueContentQuote directory.
Are you talking about this or something else?
Thanks